store.go

```diff
@@ -37,7 +37,8 @@ func (c *controller) closeStores() {
 	}
 }
 
-func (c *controller) getStore(scope string) datastore.DataStore {
+// GetStore returns the datastore for the given scope.
+func (c *controller) GetStore(scope string) datastore.DataStore {
```

Comment:
you're right. For some reason, I thought `drivers.go` was in a different package. I reverted this change.